pymchelper/readers/shieldhit.py

```diff
@@ -150,7 +150,8 @@ def test_version_0p6(self):
             # if magic string is present, deep check for version number
             if sh_bdo_magic_number == x['magic'][0]:
                 vmaj, vmin = x['vstr'][0].decode('ASCII').split('.')
-                return (sh_bdo_magic_number == x['magic'][0]) and (int(vmaj) >= 0) and (int(vmin) >= 6)
+                fver = float(vmaj) + 0.1 * float(vmin)  # build version number as a floating point number
```

Comment:
According to semantic versioning vmin can be an integer greater than 10.
Why not to use version comparison available in python standard library ?
http://stackoverflow.com/questions/11887762/compare-version-strings

See i.e.:
```
>>> from distutils.version import LooseVersion, StrictVersion
>>> LooseVersion("2.3.1") < LooseVersion("10.1.2")
True
>>> StrictVersion("2.3.1") < StrictVersion("10.1.2")
True
```

Use any of this is fine as we have only integer-versions here (do we?).